backend/src/main/resources/templates/api-call.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ody Admin Page</title>
+    <link rel="stylesheet" type="text/css" href="css/api-call.css">
+    <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
+</head>
+<body>
+<div class="container">
+    <h1>오디 어드민 페이지</h1>
+
+    <br>
+    <h2>외부 API 호출 횟수 조회하기</h2>
+    <table class="data-table">
+        <thead>
+        <tr>
+            <th></th>
+            <th>Odsay API</th>
+            <th>Google API</th>
```

Comment:
[제안] 단위를 명시해주면 어떨까요?
```suggestion
            <th>Odsay API(일별)</th>
            <th>Google API(월별)</th>
```